### DISCUSSION TITLE
Python vs NoPython

### DISCUSSION THREAD

**Discussion by TheTechnicalSpecialist** [grahampugh/erase-install#189] [Q&A]
I see the intaller comes with a Python and a noPython version.

I can't find any docs on when to consider python installation vs nopython installation.

Can anyone explain the differences and when to consider which version to use?

**Comment by grahampugh** [comment]
I would normally recommend the standard `erase-install` package (or the DEPNotify package, which also includes the python distribution). The `nopython` package is suitable if you are already deploying a compatible python3 distribution via some other means.

**Reply by TheTechnicalSpecialist** [reply]
Thanks. Makes sense!
